.drone.yml

```diff
@@ -10,7 +10,7 @@ trigger:
 
 steps:
   - name: setup
-    image: abakus/lego-testbase:python3.11
+    image: abakus/lego-testbase:latest
```

Comment:
I thought we agreed that `latest` made the most sense? The image also has `coverage` installed. It literally has `lego` in its name, and it is tailored for this repository. I don't see the point in using anything but latest?

Reply:
because you should never use latest tags. If we push a new image we break stuff. It's better to just use a proper tag

Reply:
Why would we just randomly push a new image? Of course that's going to break stuff? `lego-testbase` is for `lego`?

Reply:
because that's what people do. Yes, but it's very easy to inadvertently break something. It may not very likely, but it's just really bad practice to use `latest`.

Reply:
well, it's not like a faulty docker push is going to break anything critical - only a CI pipeline for some minor time.
But sure, if you really want to, we can rename it, but to what? I'm not sure about `python3.11-poetry` because it also includes `codecov`. Maybe also include the `poetry` version?

Reply:
Personally I think just `3.11` or `py3.11` or `python3.11` is good enough. We are not likely to break stuff if we change other things.

Reply:
And now `drone` refuses to pull the updated image ... 😩